client/scss/components/_bulk_actions.scss

```diff
@@ -85,7 +85,7 @@
     }
 
     .num-objects-in-listing {
-      color: $color-teal-light;
+      color: $color-white;
```

Comment:
I believe I went for white because light teal colors aren’t intended to be used for text like this (just backgrounds and borders). And the previous iteration was also indistinguishable from white. 

You’re right though, it’s probably a lesser evil to use a light teal for this until we completely revisit this button.